keps/2724-topology-aware-schedling/README.md

```diff
@@ -463,28 +471,55 @@ at each topology level to the specific subset of nodes.
 type PodSetAssignment struct {
   ...
 
-  // TopologyAssignment indicates the resources assigned per topology level
+  // topologyAssignment indicates the topology assignment divided into
+  // topology domains corresponding to the lowest level of the topology.
+  // The assignment specified the number of Pods to be scheduled per topology
+  // domain.
+  //
+  // Example:
+  //
+  // topologyAssignment:
+  //   levels:
+  //   - cloud.provider.com/topology-block
+  //   - cloud.provider.com/topology-rack
+  //   domains:
+  //   - values: [block-1, rack-1]
+  //     count: 4
+  //   - values: [block-1, rack-2]
+  //     count: 2
+  //
   // +optional
   TopologyAssignment *TopologyAssignment `json:"topologyAssignment,omitempty"`
 }
 
 type TopologyAssignment struct {
-  // Groups contains the list of assignments split into groups corresponding
-  // to the same topology domain at the lowest level of the hierarchy.
+  // domains is a list of topology assignments split by topology domains at
+  // the lowest level of the topology.
+  //
   // +required
-  // +listType=atomic
-  // +kubebuilder:validation:MinItems=1
-  Groups []TopologyAssignmentGroup `json:"groups"`
+  Domains []TopologyDomainAssignment `json:"domains"`
+
+  // levels is an ordered list of keys denoting the levels of the assigned
+  // topology (i.e. node label keys), from the highest to the lowest level of
+  // the topology.
+  //
+  // +required
+  Levels []string `json:"levels"`
 }
 
-type TopologyAssignmentGroup struct {
-  // NodeLabels constitutes the nodeSelector for a given slice of pods. It
-  // defines values for all labels configured in the Topology.Levels.
-  // +kubebuilder:validation:MinItems=1
-  NodeLabels map[string]string `json:"nodeLabels"`
+type TopologyDomainAssignment struct {
+  // values is an ordered list of node selector values describing a topology
```

Comment:
It would be clearer if you duplicated this description in TopologyAssignment.

Reply:
Sure, I've updated the TopologyAssignment description to include the information on how the node selector is specified. PTAL.